pkg/plugins/runtime/gateway/testdata/http/no-timeout.yaml

```diff
@@ -25,7 +25,7 @@ Clusters:
         envoy.extensions.upstreams.http.v3.HttpProtocolOptions:
           '@type': type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
           commonHttpProtocolOptions:
-            idleTimeout: 0s
+            idleTimeout: 3600s
```

Comment:
Why is this changing?

Reply:
This value changes because the gateway code is setting a different timeout. Because MeshTimeout is a new default it overrides the timeouts set by gateway code. There is no way (yet) to create only a policy that targets all gateways